Extract/Extract.ipynb

```diff
@@ -0,0 +1,264 @@
+{
```

Comment:
Let's give the whole thing a title and a short description of what this notebook does.

Reply:
Also, 'Extract' is a pretty bad name. The folder name should say something about what this notebook is doing.

Reply:
Gave short description changed folder name to MAXQDA-Conversion